strongbox-web-core/src/main/java/org/carlspring/strongbox/security/authentication/StrongboxAuthenticationFilter.java

```diff
@@ -42,45 +45,37 @@ protected void doFilterInternal(HttpServletRequest request,
             throws ServletException,
                    IOException
     {
-        Authentication result = null;
-
-        for (final Authenticator authenticator : authenticatorsRegistry)
-        {
-
-            final AuthenticationSupplier authenticationSupplier = authenticator.getAuthenticationSupplier();
+        Authentication authentication = authenticationSuppliers.supply(request);
 
-            logger.debug("Authentication supplier attempt using {}", authenticationSupplier.getClass()
-                                                                                           .getName());
-            final Authentication authentication = authenticationSupplier.supply(request);
+        if (authentication == null) {
```

Comment:
@fuss86,

Could you please re-indent this (and any other blocks) according to the coding convention? Thanks!